MapleServer2/PacketHandlers/Game/MoveFieldHandler.cs

```diff
@@ -44,6 +44,7 @@ public override void Handle(GameSession session, PacketReader packet)
                     session.Player.Rotation = session.FieldPlayer.Rotation;
                     session.Player.Coord = session.Player.ReturnCoord;
                     session.Player.ReturnCoord.Z += Block.BLOCK_SIZE;
+                    DatabaseManager.UpdateCharacter(session.Player);
```

Comment:
just curious, any reason we need to save to database upon a map change?

Reply:
We don't need to save on map change but I'm using as one way to manually test if changes are being saved. I could remove these, and add a command to save and implement a task to save every 10~20 minutes?

Reply:
As long as this is the only place we are saving unnecessarily, then yeah it's fine